### ISSUE TITLE
[a11y] User email address from “Subscribe to Mozilla VPN” screen is not correctly narrated by the Screen Reader

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#5056]
**Affected versions:**

* Mozilla VPN 2.12.0 (2.202211171941); 

* Mozilla VPN 2.12.0 (2.202211171926);

**Tested Platforms:**

* MacOS 11.6;

* Windows 11;

**Prerequisites:**

* Have the Voice Over enabled (MacOS: Apple menu > System Preferences > Accessibility > VoiceOver > Check "Enable VoiceOver") or other Screen reader opened;

* Have Mozilla VPN app installed and opened;

* Be in “Subscribe to Mozilla VPN” screen;

**Steps to reproduce:**

1. Press the right arrow key on MacOS until the “We couldn’t find a subscription for user_email_address” text is in focus / Press the Caps+O keys on Windows;
1. Observe the behavior;

**Expected result:**

* The user email address is correctly narrated by the Screen Reader;

**Actual result:**

* “<bstyle=’color:#3D3D3D;” color is narrated by the Screen Reader, followed by the user email address;

**Notes:**

* See the attached video;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-3390)
┆Reporter: Bianca Hidecuti

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Lesley Norton, could you please add the fix version? Thank you.

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Verified this as fixed on the latest 2.13 builds (2.13.0 (2.202301161600)) from the release branch, using MacOS 12.6.1 and Windows 11.

User email address from “Subscribe to Mozilla VPN” screen is now correctly narrated by the Screen Reader;

Attaching postfix video.

I also added the 2.13 fix version, as the fix is merged into the release branch.

!vpn-3390.mov|width=2502,height=2036!
